README.md

```diff
@@ -29,6 +31,25 @@
 
 - Preview at: <http://localhost:8000>
 
+#### With Docker
+
+- Open up a terminal and pull the image
+  ```sh
+  docker pull squidfunk/mkdocs-material
+  ```
+
+- Run the container
+  ```sh
+  docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material
+  ```
+
+- If you are using plugins, create a DOCKERFILE to extend the official images
+    ``` Dockerfile title="Dockerfile"
+  FROM squidfunk/mkdocs-material
+  RUN pip install -r requirements.txt
+  ```
+  Make sure to have a requirements.txt file in the root folder.
+
```

Comment:
This is a generic "how to use MKDocs with Docker"-explanation.

But in this repository, the `Dockerfile` and the `requirements.txt` files are already created and _should_ be used as-is.

So these instructions can be a lot shorter/simpler.

See: #21

Reply:
True, thanks, I'll update